src/config/stats.ts

```
@@ -149,6 +149,13 @@ export const DEMO_LINKS = {
 // ===== LINK COLLECTIONS FOR TESTING =====
 // These collect all links for easy testing
 
+// Newsletter links for testing
+export const NEWSLETTER_LINKS = {
+  weeklyNewsletter: NEWSLETTER_INFO.weekly.url,
+  jsTestingTips: NEWSLETTER_INFO.jsTestingTips.url,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)